qa/full-cluster-restart/build.gradle

```diff
@@ -19,3 +19,8 @@ BuildParams.bwcVersions.withIndexCompatible { bwcVersion, baseName ->
     systemProperty("tests.old_cluster_version", bwcVersion)
   }
 }
+
+tasks.withType(Test).configureEach {
```

Comment:
@breskeby will this turn off parallelism for all test tasks everywhere, or just test tasks in this module?

Reply:
just all test tasks within this `full-cluster-restart` module